pynest/examples/Potjans_2014/helpers.py

```diff
@@ -293,7 +292,8 @@ def firing_rates(path, name, begin, end):
         # zeros are included
         all_mean_rates.append(np.mean(rate_per_neuron))
         all_std_rates.append(np.std(rate_per_neuron))
-    print('Mean rates: {} spikes/s'.format(np.around(all_mean_rates, decimals=3)))
+    print('Mean rates: {} spikes/s'.format(np.around(all_mean_rates,
+                                                     decimals=3)))
```

Comment:
Could you revert back to one line for this? We allow lines of 120 characters in NEST.

Reply:
Done